index.html

```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
+              <p class="portfolio-title">Multi-Post Stories</p>
+              <ul class="client-role-year">
+                <li class="client">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="actions">
+              <button class="action action-text btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="top">
+          <p class="about-my-self-title">About Myself</p>
+          <p class="about-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="social-media">
+            <h2 class="lets-connect">LET’S CONNECT</h2>
+            <ul class="social-icon">
+              <li>
+                <a href="#"> <img src="icons/twitter.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/Linkedin icon.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/m.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/github.svg" /> </a>
+              </li>
+              <li>
+                <a href="#"> <img src="icons/peace.svg" /> </a>
+              </li>
+            </ul>
+          </div>
+          <div class="actions">
+            <button class="action action-text btn">Get my resume</button>
+          </div>
+        </div>
+        <div class="skill-list">
+          <div class="languages">
+            <div class="language-title title-arrow">
+              <span> Languages </span>
+              <span> <img src="icons/down-arrow.png" alt="" /> </span>
+            </div>
+            <hr class="separator" />
+            <div class="language-wrapper">
+              <ul class="language-list">
+                <li class="language">
+                  <span class="langauge-logo">
+                    <img src="assets/images/js.png" alt="js" />
+                  </span>
+                  <span class="language-name"> Javascript </span>
+                </li>
+                <li class="language">
+                  <span>
+                    <img src="assets/images/html elipse.png" alt="html" />
+                  </span>
```

Comment:
- [ ] to match Figma design kindly add some paddings/margins between the skill icon and the container of the skill , and kindly don't forget to add the bottom border line below each one of them

**_figma design :_**
![image](https://user-images.githubusercontent.com/89282221/198538042-f5422327-6fca-46e4-b39c-b931421ea680.png)